src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -0,0 +1,201 @@
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.WhoIsAtRisk
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.oasysarnapi.RiskSummaryResponse
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.exception.ClientTimeoutException
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.exception.PersonNotFoundException
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+import java.util.Locale
+
+@Service
+class RiskService(
+  @Qualifier("communityApiClientUserEnhanced") private val communityApiClient: CommunityApiClient,
+  @Qualifier("assessRisksNeedsApiClientUserEnhanced") private val arnApiClient: ArnApiClient
+) {
+  suspend fun getRisk(crn: String): RiskResponse {
+    val userAccessResponse = getValue(communityApiClient.getUserAccess(crn))
+    return if (true == userAccessResponse?.userExcluded || true == userAccessResponse?.userRestricted) {
+      RiskResponse(userAccessResponse = userAccessResponse)
+    } else {
+      val riskSummaryResponse = getValue(arnApiClient.getRiskSummary(crn))
+      val personalDetailsOverview = fetchPersonalDetails(crn)
+      val riskOfSeriousHarm = extractRiskOfSeriousHarm(riskSummaryResponse!!)
+      val natureOfRisk = extractNatureOfRisk(riskSummaryResponse)
+      val whoIsAtRisk = extractWhoIsAtRisk(riskSummaryResponse)
+      val circumstancesIncreaseRisk = extractCircumstancesIncreaseRisk(riskSummaryResponse)
+      val factorsToReduceRisk = extractFactorsToReduceRisk(riskSummaryResponse)
+      val whenRiskHighest = extractWhenRiskHighest(riskSummaryResponse)
+      val mappa = handleFetchMappaApiCall(crn)
+      val predictorScores = null // TODO Andrew's API will provide this
+      val contingencyPlan = null // TODO Andrew's API will provide this
+
+      return RiskResponse(
+        personalDetailsOverview = personalDetailsOverview,
+        riskOfSeriousHarm = riskOfSeriousHarm,
+        mappa = mappa,
+        predictorScores = predictorScores,
+        natureOfRisk = natureOfRisk,
+        contingencyPlan = contingencyPlan,
+        whoIsAtRisk = whoIsAtRisk,
+        circumstancesIncreaseRisk = circumstancesIncreaseRisk,
+        factorsToReduceRisk = factorsToReduceRisk,
+        whenRiskHighest = whenRiskHighest
+      )
+    }
+  }
+
+  private suspend fun handleFetchMappaApiCall(crn: String): Mappa? {
+    return try { fetchMappa(crn) } catch (e: WebClientResponseException.NotFound) {
+      log.info("No MAPPA details available for CRN: $crn - ${e.message}")
+      Mappa(level = "", isNominal = true, lastUpdated = "")
+    }
+  }
+
+  private suspend fun extractNatureOfRisk(riskSummaryResponse: RiskSummaryResponse): NatureOfRisk {
+    return NatureOfRisk(
+      description = riskSummaryResponse.natureOfRisk ?: "",
+      oasysHeading = OasysHeading(
+        number = "10.2",
+        description = "What is the nature of the risk?"
```

Comment:
What do we do with these descriptions? Just wondering if they should be stored in a config file? Or stored on the frontend?

Reply:
I talked about it with Jon, at the time we decided to hard code them in the backend.
I'm happy to incline to the FE / a config file. Do you have a preference Jack?

Reply:
Ok if you're happy then I'm happy.

Reply:
If at some point we start to think about introducing a Welsh language, we might want to think about moving this to the frontend.